src/Uno.UI.Runtime.Skia.Wpf/Rendering/RenderSurfaceType.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Uno.UI.Skia;
```

Comment:
@jeromelaban Is this breaking change okay? If so, I'll doc it as well. Otherwise, I'll revert.